samples/bluetooth/direction_finding_connectionless_rx/src/main.c

```diff
@@ -194,6 +204,8 @@ static void scan_recv(const struct bt_le_scan_recv_info *info,
 	       info->interval, info->interval * 5 / 4, info->sid);
 
 	if (!per_adv_found && info->interval != 0) {
+		sync_create_timeout_ms =
+			adv_interval_to_ms(info->interval) * SYNC_CREATE_TIMEOUT_INTERVAL_NUM;
```

Comment:
If `adv_interval_to_ms` is only used to do this calculation, we could consider actually doing the entire calculation in a function, instead of just doing part of it like here.

Reply:
multiplication is now part of a function `sync_create_timeout_get`.